message_parser/src/main/java/cumulus_message_adapter/message_parser/MessageAdapter.java

```diff
@@ -47,6 +50,16 @@ public String CallMessageAdapterFunction(String messageAdapterFunction, String i
             writer.write(inputJson);
             writer.close();
 
+            // Honestly i'm not sure why scanner doens't work below, i think the
+            // issue is taht this needs to be consumed before the process will
+            // exit (too large for processbuilder?)
```

Comment:
Suggest deleting the above and replacing with something succinct like:

```suggestion
            // fix to avoid overflowing processBuilder InputStream
```